src/pages/community/foundation/local-initiative.mdx

```diff
@@ -0,0 +1,136 @@
+are currently out-of-scope of GraphQL Foundation support. These other events
+shall not be referred to as "official", but are still encouraged to follow the
+guidelines that are applicable (in particular, the Code of Conduct and Brand
+Guidelines must still be honoured).
+
+## Support
+
+Each approved Local is entitled to request the following support for each
+event:
+
+- **Sponsorship** up to USD $200 per event (max 1 per quarter) for approved
+  costs (see below). More sponsorship might be available directly from
+  individual foundation members, see details below.
+- **Swag bundle**\* to distribute to speakers, attendees, or organizers (max 1
+  per quarter).
+- **Google Slides templates** for:
+  - Code of Conduct
+  - Call for Papers (CFP)
+  - Sponsors
+  - Schedule
+  - Speaker/talk introduction
+- **Promotion** through GraphQL’s Twitter and LinkedIn channels.
+- **Custom artwork** adhering to the GraphQL Brand Guidelines; including
+  avatars, event banners, and slide templates. \*\*
+
+\* The content of swag bundles may change from time to time. At time of writing
+they are likely to consist of 2-3 t-shirts (sizes to be chosen by organizer),
+stickers, pin-badges, and a GraphQL bow-tie!
+
+\*\* There may be a significant delay on the delivery of custom artwork
+currently, we're still figuring out designs.
+
+## Approved Costs
+
+Sponsorship funds may be used towards:
+
+- Food (be sure to consider dietary restrictions and allergies)
+- Non-alcoholic drinks (alcoholic beverages require sponsor approval or should
+  be paid out of alternative funds)
+- Venue hire
+- Swag for speakers or attendees
+- Promotional materials
+- AV resources (e.g., camera stand, microphone)
+
+If your GraphQL Local application is approved you will be provided with
+invoicing instructions.
+
+## Caveats
+
+All support is subject to the finite resources (both financial and temporal) of
+the GraphQL Local initiative. Sponsorship may be adjusted both up and down based
+on expected attendance and is discretionary. Bespoke work, e.g. custom artwork,
+may have long and unpredictable lead times. Your application may be declined for
+any reason, but in particular if we think it is likely that the initiative will
+exhaust its budget we will decline all further applications until more budget
+becomes available.
+
+If you'd like to help support the GraphQL Foundation's various programs and
+initiatives, consider becoming a GraphQL Foundation member.
+
+## Additional Support
+
+If you need extra support for your event (e.g., transportation costs or higher
+venue fees/larger food and drinks budget for larger get-togethers), individual
+GraphQL Foundation member organizations may be able to offer additional support,
+often with a small ask such as saying a few words about them during the event.
+If your gathering is approved, you will be provided with contact information for
+member organizations who have expressed interest in supporting gatherings.
+
+## Apply now!
```

Comment:
Do we want to say "Apply" or "Sign Up"? We had been talking like this is a "sign up and you get it, but you have to adhere to these to stay on" kind of thing.

Reply:
I've modified the language, I don't think "sign up" is quite right either since there is a process albeit a short one. Let me know what you think of the alternative wording.